piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -292,16 +292,19 @@ that had the least hand therein.
 Therefore be instructed, you rulers of this land, and take warning betimes,
 and learn wisdom, before it be hid from your eyes.`'
 
```

Comment:
This whole paper of Robinson should be embedded. Starting above after where it says "Here follows a copy of William Robinson's paper...."